pages/agent/v3/tokens.md

```diff
@@ -1,100 +1,126 @@
 
 ## Using and storing tokens
 
-The token is used by the Buildkite Agent's [start](/docs/agent/v3/cli-start#starting-an-agent) command, and can be provided on the command line, set in the [configuration file](/docs/agent/v3/configuration), or provided using the [environment variable](/docs/pipelines/environment-variables) `BUILDKITE_AGENT_TOKEN`.
+An agent token is used by the Buildkite Agent's [start](/docs/agent/v3/cli-start#starting-an-agent) command, and can be provided on the command line, set in the [configuration file](/docs/agent/v3/configuration), or provided using the [environment variable](/docs/pipelines/environment-variables) `BUILDKITE_AGENT_TOKEN`.
 
 It's recommended you use your platform's secret storage (such as the [AWS Systems Manager Parameter Store](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-paramstore.html)) to allow for easier rollover and management of your agent tokens.
 
-## Creating tokens
-
-New tokens can be created using the [GraphQL API](/docs/apis/graphql-api) with the `agentTokenCreate` mutation.
-
-For example:
-
-```graphql
-mutation {
-  agentTokenCreate(input: {
-    organizationID: "organization-id",
-    description: "A description"
-  }) {
-    tokenValue
-    agentTokenEdge {
-      node {
-        id
-      }
-    }
-  }
-}
-```
+## Create a token
+
+New agent tokens can be created either using the [_Agent Tokens_ page of a cluster](#create-a-token-using-the-buildkite-interface), or via the [REST API's create agent token](#create-a-token-using-the-rest-api) feature.
+
+> 📘 An agent token's value is only displayed once
+> As soon as the agent token's value is displayed, copy its value and save it in a secure location.
+> If you forget to do this, you'll need to create a new token to obtain its value.
+
+It is possible to create multiple agent tokens (for your Default cluster or any other cluster in your Buildkite organization) using the processes described in this section.
+
+### Using the Buildkite interface
+
+To create an agent token for a cluster using the Buildkite interface:
+
+1. Select _Agents_ to access the _Agent Clusters_ page.
+1. Select the cluster containing the agent token to revoke.
+1. Select _Agent Tokens_ and on this page, select _New Token_.
+1. In the _Description_ field, enter an appropriate description for the agent token.
+
+    **Note:** The token description should clearly identify the environment the token is intended to be used for (for example, `Read-only token for static site generator`), and is listed on the _Agent tokens_ page of your specific cluster the agent connects to. This page can be accessed by selecting _Agents_ > the specific cluster tile > _Agent Tokens_.
```

Comment:
Is this why this is important? ie, does this wording change make this read more clearly (or did I misinterpret what you were hoping to get across?)

```suggestion
    **Note:** The token description should clearly identify the environment the token is intended to be used for (for example, `Read-only token for static site generator`), as it is listed on the _Agent tokens_ page of your specific cluster the agent connects to. This page can be accessed by selecting _Agents_ > the specific cluster tile > _Agent Tokens_.
```

Reply:
Great pick-up!